qbft/spectest/tests/proposal/prepared_previously_justification.go

```diff
@@ -38,7 +41,7 @@ func PreparedPreviouslyJustification() *tests.MsgProcessingSpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "previously prepared proposal",
 		Pre:           pre,
```

Comment:
is this a duplicate of current_round_prev_prepared?

Reply:
I think it is, so I'm removing `CurrentRoundPrevPrepared` (duplicate of `PreparedPreviouslyJustification`) and `CurrentRoundPrevNotPrepared` (duplicate of `NotPreparedPreviouslyJustification`)